scripts/auto_sshfs_mount.py

```diff
@@ -41,10 +41,11 @@ def AttemptUnmount(local):
             pass
         
 #==============================================================================
-#==============================================================================
 
-if __name__ == '__main__':
+def MountSSHFS():
+    '''
     
+    '''
     #Specify the mount point as ~/Huntsmans-Pro (for huntsman user)
     home = os.path.expandvars('$HOME')
     local = os.path.join(home, 'huntsman', 'Huntsmans-Pro')
```

Comment:
I'm still unclear why you mount as `~/Huntsman-Pro`, requiring a symlink, rather than just mounting directly at `$PANDIR/images`.

Reply:
We could do this later. But at the moment I think its more flexible to have the whole filesystem mounted in case we end up using it for something other than synching images.